dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -633,3 +633,19 @@ ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host n
 ssl.client.handshake.error.CWPKI0824E.explanation=An error occurred during the SSL handshake.  Host name verification is enabled and server's identity can not be verified.  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.
 ssl.client.handshake.error.CWPKI0824E.useraction=When host name verification is enable the server's being accessed needs to be setup with a certificate contain the proper information needed to verify the server's identity.
 # -------------------------------------------------------------------------------------------------
+ssl.cacerts.error.CWPKI0825W=CWPKI0825W: The trustDefaultCerts attribute is enabled but the default truststore failed to initialize. Because the default truststore did not initialize, only the truststore that is configured by the {0} SSL configuration is used. The exception that is returned is: {1}. 
+ssl.cacerts.error.CWPKI0825W.explanation=The default truststore did not initialize because it could not be retrieved from the trust manager. The default truststore is not used for trust. Only the configured SSL configuration is used.
+ssl.cacerts.error.CWPKI0825W.useraction=Ensure that the default truststore file exists and is accessible
+# -------------------------------------------------------------------------------------------------
+ssl.environment.cert.error.CWPKI0826W=CWPKI0826W: The certificate was not retrieved from the {0} environment variable. The certificate was not added to the {1} truststore.  The exception that is returned is: {2}.
+ssl.environment.cert.error.CWPKI0826W.explanation=The certificate that is specified in the environment variable cannot be accessed and is not included in the truststore. This might be due to a problem with the environment variable, the file name, or the certificate.
```

Comment:
**change**
The certificate that is specified in the environment variable cannot be accessed and is not included in the truststore. This might be due to a problem with the environment variable, the file name, or the certificate.
**to**
The certificate that is specified in the environment variable cannot be accessed and is not included in the truststore. This problem might be caused by the environment variable, the file name, or the certificate. A failure to add the certificate does not invalidate the SSL configuration, but it might cause problems with establishing trust for outbound connections.